lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -627,6 +636,7 @@ def main():
 
     option_minimal_versions = dict(
         cache_from=dict(docker_py_version='2.1.0', docker_api_version='1.25'),
```

Comment:
@felixfontein it may be great if docs can include these version requirements somehow, currently there's none

![image](https://user-images.githubusercontent.com/1430856/50441132-33971700-0934-11e9-9132-cc851759da27.png)

Reply:
Indeed. Currently, the only way to do that would be to add that information manually, which is somewhat annoying and error-prone. Could you create a Feature Request issue for this (for the docker modules)? Then we can collect ideas how we can improve this in the future.

Reply:
Opened #50319